drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,6 +32,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include <zephyr/irq.h>
 #include <zephyr/net/lldp.h>
 #include <zephyr/drivers/hwinfo.h>
+#include <zephyr/sys/crc.h>
```

Comment:
Not required, this header is already included as the function is already used in this driver

Reply:
ok